.github/workflows/ci.yml

```diff
@@ -36,6 +36,8 @@ jobs:
       # Test some linkerd crds
       - run: just test-linkerd-serverauth
       - run: just test-linkerd-server
+      # Test stdout of commands
+      - run: just test-trycmd-verify
```

Comment:
presumably this is faster to run that the `just test-X` invocations that needs rebuilds, so maybe put this before the integration tests

Reply:
ok moved above the text-X invocations. The trycmd test runs in less than 1 second on my machine